doc/source/api/general/data_reader.rst

```diff
@@ -0,0 +1,64 @@
+.. _ref_data_reader:
+
+DataFile
+========
+
+The DataFile class provides a reader for Fluent data files.
+
+Sample usage
+------------
+
+You can use the DataFile class by importing it and passing a data file path.
+For some case file dependent features we need to pass the case file handle from
+the CaseFile class as well as shown in the code snippet.
```

Comment:
the CaseFile class as shown in the following code snippet.

Reply:
Thank you